.rubocop.yml

```diff
@@ -8,6 +8,7 @@ AllCops:
     - "*/bin/**/*"
     - "**/tmp/**/*"
     - "*/spec/fixtures/**/*"
+    - "dry-run/**/*"
```

Comment:
🔎 Can you provide context on this change? It doesn't _seem_ related to the Gemfile.

Reply:
Oh, absolutely, I should've at least extracted that to its own commit, let me do that. That's because `dry-run` is a common ignored folder at the repo root generated by our `dry-run.rb` script. If you run RuboCop from the repo root, you don't want to inspect all the repos in there.